projects/arc-docs/src/app/docs/auth-doc/components/configure-token-doc/configure-token-doc.component.html

```diff
@@ -0,0 +1,90 @@
+<div class="intro-wrapper">
+  <div class="1st-phase">
+    <nb-card>
+      <nb-card-header fixed>
+        <h2 class="nb-card-header-title">
+          Receiving user token after login/registration
+        </h2>
+      </nb-card-header>
+      <nb-card-body>
+        <ul>
+          <li>
+            At this step, it's time to get a user token. Primarily, we need a
+            code. With the help of this code, we call the getToken API, which
+            returns a token after a successful OAuth 2.0 authentication. This
+            token will allow us to communicate with the server and, for
+            instance, display the username in the header of the application.
```

Comment:
```suggestion
            At this step, we need to authenticate the user and get users JWT token. We follow OAuth 2.0 authentication mechanism. Here we first request for code and using that code we generate the required token. This token will be used for further requests as Authorization header.
```

Reply:
sorry ? i don't understand what changes you want in this

Reply:
Changes are suggested

Reply:
ok done